backgroundless-pins.js

```diff
@@ -65,8 +73,32 @@ export class BackgroundlessControlIcon extends ControlIcon {
         this.icon.tint = Number.isNumeric(this.tintColor) ? this.tintColor : 0xffffff;
         return this;
     }
+
+    async drawV11() {
+        // Don't draw a destroyed Control
+        if (this.destroyed) return this;
+
+        // Load the icon texture
+        this.texture = this.texture ?? await loadTexture(this.iconSrc);
+
+        // Set the icon texture
+        this.icon.texture = this.texture;
+
+        // Set the icon width and height
+        this.icon.width = this.icon.height = this.size;
+
+        // Hide the background
+        this.bg.visible = false;
+
+        // Refresh
+        return this.refresh();
+    }
+}
+
+function isv11OrLater() {
+    return game.data.version >= 11;
```

Comment:
Why don't you use the same source as `isV10OrLater`? I had to patch it like this to make it work on my install

```suggestion
    return game.release.generation >= 11;
```